File: codyze_legacy/src/test/resources/mark_java/simple_instancestring.java
class bla {

  java.lang.String call(java.lang.String s) {
    return "";
  }


  int main() {
    java.lang.String s = "AES/CBC/123";
    java.lang.String j;

    j = call(s);

    bla b = new bla();
    j = b.call(s);
  }
}

File: codyze_legacy/src/test/resources/mark_cpp/simple_instancestring.cpp
#include <iostream>

class Test {

public:
    std::string call(std::string s) {
      return "";
    }
}


int main() {
  std::string s = "AES/CBC/123";
  std::string j;

  Test t();
  j = t.call(s);
}

